examples/todo/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@loopback/example-todo",
-  "version": "1.7.4",
+  "version": "1.8.0",
```

Comment:
We are bumping up version numbers during the release process, please revert this line.

Reply:
Will do -- but note that it will cause CI to fail

Reply:
That's weird and not expected at all. Can you try to rebase your PR on top of the latest master please?

Reply:
Oh, oops ... forgot to switch git reflexes to fork mode and thought I was already up to date there... I suspect that will do the trick